addon/services/flash-messages.js

```diff
@@ -68,6 +68,13 @@ export default Service.extend({
     return this;
   },
 
+  getFlashObject() {
```

Comment:
IMO `getLastFlashObject` holds more meaning than `getFlashObject`.

Or since we are talking about a queue of message, why not provide accessors such as `peekFirst` and `peekLast` ?